src/features/abi-methods/components/decoded-abi-method-arguments.tsx

```diff
@@ -5,16 +5,20 @@ import { TransactionLink } from '@/features/transactions/components/transaction-
 import { AccountLink } from '@/features/accounts/components/account-link'
 import { ApplicationLink } from '@/features/applications/components/application-link'
 import { AssetIdLink } from '@/features/assets/components/asset-link'
+import { asAbiMethodArgumentRender } from '@/features/abi-methods/mappers'
+import { sum } from '@/utils/sum'
 
 export function DecodedAbiMethodArguments({ method }: { method: AbiMethod }) {
+  const argumentsRender = useMemo(() => method.arguments.map((argument) => asAbiMethodArgumentRender(argument)), [method.arguments])
+
   const components = useMemo(
     () =>
-      method.arguments.map((argument) => {
+      argumentsRender.map((argument) => {
         if (argument.type === AbiType.Transaction) {
           return (
             <>
-              <span>{argument.name}: </span>
-              <TransactionLink className="text-primary underline" transactionId={argument.value}>
+              <span className="text-abi-keys">{argument.name}: </span>
+              <TransactionLink className={'text-primary underline'} transactionId={argument.value}>
```

Comment:
```suggestion
              <TransactionLink className="text-primary underline" transactionId={argument.value}>
```
The same comment applied for other `className={'`